Add import globbing docs (#48)
crate/cloud-docs: docs/reference/overview.rst

CrateDB Cloud allows convenient imports directly from S3-compatible storage. 
To import a file form bucket, provide the name of your bucket, and path to
the file. The S3 Access Key ID, and S3 Secret Access Key are also needed. You 
can also specify the endpoint for non-AWS S3 buckets. Keep in mind that you may

> be charged for egress, depending on your provider. There is also a limit of 10

Reviewer: ```suggestion
be charged for egress traffic, depending on your provider. There is also a volume limit of 10
```

> GiB for S3 imports. The usual file formats are supported.

Reviewer: Is it "per-file", actually?
```suggestion
GiB per file for S3 imports. The usual file formats are supported.
```

> GiB for S3 imports. The usual file formats are supported.

Reviewer: > The usual file formats are supported.

Which ones?

can also specify the endpoint for non-AWS S3 buckets. Keep in mind that you may
be charged for egress traffic, depending on your provider. There is also a
volume limit of 10 GiB per file for S3 imports. The usual file formats are

> supported - CSV, Json, and Parquet.

Reviewer 2: ```suggestion
supported - CSV (all variants), Json (JSON-Lines, JSON Arrays and JSON Documents), and Parquet.
```

        }]
        }

.. _overview-cluster-import-globbing:

> Import globbing

Reviewer 3: ```suggestion
Importing multiple files
```

> ~~~~~~~~~~~~~~~

Reviewer: Globbing is a pretty technical term. Maybe use "wildcard" instead?

> 

Reviewer: What about such a little introductory paragraph before going into the technical details?

> You can import multiple files from an S3 bucket at once by using the well-known
> wildcard notation, also known as "globbing". In computer programming, [glob patterns]
> specify sets of filenames with [wildcard characters].

[glob patterns]: https://en.wikipedia.org/wiki/Glob_(programming
[wildcard characters]: https://en.wikipedia.org/wiki/Wildcard_characters

        }]
        }

.. _overview-cluster-import-azure:

> Azure Container Storage
> ^^^^^^^^^^^^^^^^^^^^^^^

Reviewer 3: ```suggestion
Import from Azure Blob Storage Container
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

> Importing from Azure Storage Containers is also supported. In this case, the
> secret consists of a secret name, an Azure Storage Connection string or an
> Azure SAS Token URL.
> 
> There is an option to import data from a private Azure Blob Storage container
> using a stored secret. Secret can be added by an admin user at the organization
> level.

Reviewer 3: this seems to be a duplicate. maybe something like this ?

```suggestion
Importing data from private Azure Blob Storage containers is possible using a stored secret, which includes a secret name and either an Azure Storage Connection string or an Azure SAS Token URL. An admin user at the organization level can add this secret.
```
